stable/cockroachdb/README.md

```diff
@@ -154,7 +154,7 @@ pulling up the CockroachDB admin UI by port-forwarding from your local machine
 to one of the pods (replacing "release-cockroachdb-0" with one of your pods' names):
```

Comment:
We should also change "release-cockroachdb-0" on this line to "my-release-cockroachdb-0".